Easy_Problems/Coderbyte15.js

```diff
@@ -0,0 +1,12 @@
+function Palindrome(str) {
```

Comment:
This doesn't make sense. Why are you using a for loop and never using the variable i? And why are there so many splits and joins?